internal/util/vault_tokens_test.go

```diff
@@ -110,3 +111,50 @@ func TestInitVaultTokensKMS(t *testing.T) {
 		t.Errorf("unexpected error: %s", err)
 	}
 }
+
+// TestStdVaultToCSIConfig converts a JSON document with standard VAULT_*
+// environment variables to a vaultTokenConf structure.
+func TestStdVaultToCSIConfig(t *testing.T) {
+	vaultConfigMap := `{
+		"KMS_PROVIDER":"test",
```

Comment:
Eventhough it does not matter, isnt it better to keep a test KMS name with a string `vault`, so that tomorrow we can have another test with other KMS provider and consistency ? for exam: `aws-kms-test`...etc?

Reply:
I'll do that. The `KMS_PROVIDER` is not used once it has been parsed, I think. This parsing is part of the VaultTokensKMS, so the provider is already selected.